free-3.md

```diff
@@ -57,12 +87,19 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I don't think there will be a lot of language decisions -- the input I need only requires 4 basic pieces of information from the user (key signature, clef, tempo in BPM, and time signature). None of the semantics of the language will look very code-y at all! 
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+I don't think this idea is too ambitious -- it's very easy to determine what keys are pressed and for how long they are pressed, and the time that the key is pressed can be compared to the designated BPM and key signature to determine the duration of the note relative to the BPM (tempo). The tricky part would be to translate that into a visual representation of the notes.
 
```

Comment:
I wonder how you would be able to integrate your DSL with MuseScore. Do you know if MuseScore allows for developers to add plugins, like the language you are supposing? If not, it seems like it would be very difficult to create a new music editor that could real time interpret the notes you type, and that might be outside the scope of this class. Chords to not strike me as an issue, since you could just hold down several keys at once. I do wonder, though, how you would be able to reach across different octaves. If all the notes were contained in a single octave, this would limit your DSL to very simple melodies, but I think it would will have a good purpose.